source/grid/grid_in.cc

```diff
@@ -3844,7 +3844,7 @@ namespace
                      ++j)
                   boundary_line.vertices[j] =
                     cell.vertices[cell_type.face_to_cell_vertices(
-                      deal_face_n, j, 0)];
+                      deal_face_n, j, numbers::default_geometric_orientation)];
```

Comment:
This denim has value 1 and not 0 though? Why is this correct.

Reply:
It isn't visible in the GitHub view but that is documented a few lines up:

https://github.com/dealii/dealii/blob/b7fcb266cc687a7c8b0fcb42663558d72aec99f0/source/grid/grid_in.cc#L3833-L3835

The orientations of subcell objects passed to `create_triangulation()` are arbitrary since we set orientations based on cell data (e.g., the first time the line appears in a cell sets its canonical orientation).

Reply:
The person who originally wrote zero was me; I have no idea why I did that nearly 5 years ago.